platform/iphone/detect.py

```diff
@@ -162,50 +152,15 @@ def configure(env):
     # Temp fix for ABS/MAX/MIN macros in iPhone SDK blocking compilation
     env.Append(CCFLAGS=["-Wno-ambiguous-macro"])
 
-    ## Link flags
-
-    if env["arch"] == "x86" or env["arch"] == "x86_64":
-        arch_flag = "i386" if env["arch"] == "x86" else env["arch"]
-        env.Append(
-            LINKFLAGS=[
-                "-arch",
-                arch_flag,
-                "-isysroot",
-                "$IPHONESDK",
-                "-Xlinker",
-                "-objc_abi_version",
-                "-Xlinker",
-                "2",
-                "-F$IPHONESDK",
-            ]
-        )
-    elif env["arch"] == "arm":
-        env.Append(LINKFLAGS=["-arch", "armv7", "-Wl,-dead_strip"])
-    if env["arch"] == "arm64":
-        env.Append(LINKFLAGS=["-arch", "arm64", "-Wl,-dead_strip"])
-
-    env.Append(
-        LINKFLAGS=[
-            "-isysroot",
-            "$IPHONESDK",
-        ]
-    )
```

Comment:
Nice cleanup if we don't need those anymore, though might be worth doing as a separate commit if it's not related to Volk specifically. Might also be something that can be backported to `3.x`?

Reply:
It can be back ported. It was never needed.